integrationtests/src/test/java/org/axonframework/integrationtests/testsuite/student/MultiEntityCommandHandlingComponentTest.java

```diff
@@ -22,6 +22,8 @@
 import org.axonframework.commandhandling.annotation.CommandHandler;
 import org.axonframework.eventhandling.EventSink;
 import org.axonframework.eventhandling.GenericEventMessage;
+import org.axonframework.eventhandling.gateway.EventAppender;
+import org.axonframework.eventhandling.gateway.EventGateway;
```

Comment:
Nit: I think the `EventGateway` import is not being used here.

Reply:
You're right. Removing